os.go

```diff
@@ -222,6 +222,16 @@ func linkCount(f os.FileInfo) string {
 	return ""
 }
 
+func getBlockSize(f os.FileInfo) (*int64, error) {
+	sysCallStat, isSysCall := f.Sys().(*syscall.Stat_t)
+	if !isSysCall {
+		return nil, fmt.Errorf("can not convert fileinfo to syscall stat")
```

Comment:
FYI, you can just return a value like `-1` if you want to return `int64` directly instead of a pointer `*int64`.

Reply:
Changed to always return an int64 directly